wiki/www/drafts.py

```diff
@@ -43,10 +41,6 @@ def get_context(context):
 					"label": _("My Contributions ") + get_open_contributions(),
 					"url": "/contributions",
 				},
-				{
-					"label": _("My Drafts ") + get_open_drafts(),
-					"url": "/drafts",
-				},
```

Comment:
didn't make sense to show drafts count on drafts page's dropdown - hence removed it